dd-sdk-android/src/main/kotlin/com/datadog/android/tracing/AndroidTracer.kt

```diff
@@ -72,6 +76,8 @@ class AndroidTracer internal constructor(
         private var random: Random = SecureRandom()
         private val logsHandler: LogHandler
 
+        private val globalTags: MutableMap<String, String> = mutableMapOf()
```

Comment:
You should use a ConcurrentHashMap here and also add a unit test for multi thread access.

Reply:
It's `val` and we set it only once, from the builder - doesn't it mean no mutations later, so thread-safety by design?

Reply:
Oh...my bad I didn't see that was only set from the Builder, yup in that case we are thread safe ;)